added Linear Gauge (#1744)
telerik/reporting-docs: user-guide/components/report-items.md

The Gauge visualizes key performance indicators (KPIs) and progress toward goals. It supports both radial (circular, like speedometers) and linear (horizontal or vertical bars) gauges. You can incorporate the Gauge in dashboards and reports where you need to visualize KPIs, progress, or performance metrics at a glance, such as sales targets, speed, or completion percentages.

### Radial Gauge

### Linear Gauge

The Linear Gauge is represented by a horizontal or vertical scale and displays its value as a percentage of the total scale's length. To add a Linear Gauge, click the **Linear Gauge Wizard** in the **Components** tab, and then use the **Configure Linear Gauge** pane to adjust the settings.

Depending on the gauge's orientation and range, you can create horizontal, vertical, single-range, and multi-range linear gauges.

> 

Reviewer: ```suggestion
```

The Gauge visualizes key performance indicators (KPIs) and progress toward goals. It supports both radial (circular, like speedometers) and linear (horizontal or vertical bars) gauges. You can incorporate the Gauge in dashboards and reports where you need to visualize KPIs, progress, or performance metrics at a glance, such as sales targets, speed, or completion percentages.

### Radial Gauge

### Linear Gauge

> The linear gauge is represented by a horizontal or vertical scale and displays its value as a a percentage of the total scale's length. By clicking the **Linear Gauge Wizard** from the **Components tray**, the **Configure Linear Gauge** pane appears on the right side:  

Reviewer: ```suggestion
The Linear Gauge is represented by a horizontal or vertical scale and displays its value as a percentage of the total scale's length. To add a Linear Gauge, click the **Linear Gauge Wizard** in the **Components** tab, and then use the **Configure Linear Gauge** pane to adjust the settings.
```

![Linear Gauge Wizard ><](images/wrd-report-items-linear-gauge-wizard.png)    

> According to the selected **Linear gauge type**, one of the following linear gauges can be created:

Reviewer: ```suggestion

The table below describes the available linear gauges:
```
